spec/controllers/teams_controller_spec.rb

```diff
@@ -184,7 +184,29 @@
 
             expect(Role.last.user).to eql existing_user
           end
+        end
+
+      #TODO: refactor this test
+      context 'selecting the conference options' do
+        let(:conference_1) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:conference_2) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:team)         { FactoryGirl.create(:team, :in_current_season) }
```

Comment:
Is it really necessary to create another (unrelated) team? If you already created a student, you could also just use this existing team I'd say:
```ruby
let(:team) { student.teams.first }
```